include/lsst/afw/geom/SeparableXYTransform.h

```diff
@@ -41,6 +41,11 @@ class Functor;
 /** @brief A 2D transform for which the pixel distortions in the in
  *  the x- and y-directions are separable.
  *
+ * @deprecated Deprecated in 14.0. Will be removed in 15.0. It is partially
+ *             replaced by ast::ParallelMap or ast::Mapping::under, but the
+ *             replacement of Functors with Mappings must be done on a
+ *             case-by-case basis.
+ *
```

Comment:
Isn't "one release" the shortest deprecation period possible? :confused: 

(Hopes DM-8969 is implemented soon...)